department-of-veterans-affairs/caseflow
# Encapsulated hearing details diff logic to fix toggle

Resolves #14117 

### Description
Fixes issue with deepDiff when calculating changes on hearings details page

### Acceptance Criteria
- [ ] User can successfully toggle back and forth between virtual/video hearings continuously

### Testing Plan
1. switch user to BVASYELLOW
1. go to hearings/schedule
1. find a video hearing (I found one on the first video hearing day in St. Petersberg)
edit hearing details
1. switch hearing type to virtual & submit modal confirm you get a success
1. switch hearing type to video & submit modal confirm you get a success
1. switch hearing type to virtual & submit modal confirm you get a success
